e2e/client/playwright/authoring.spec.ts

```diff
@@ -38,3 +38,28 @@ test('can apply "populate abstract" macro', async ({page}) => {
         page.locator(s('authoring', 'authoring-field=abstract')).getByRole('textbox'),
     ).toHaveText('test sport story body');
 });
+
+test('can embed if url is valid', async ({page}) => {
+    await restoreDatabaseSnapshot();
+
+    const monitoring = new Monitoring(page);
+
+    await page.goto('/#/workspace/monitoring');
+
+    await monitoring.selectDesk('Sports');
+
+    await page.locator(
+        s('monitoring-group=Sports / Working Stage', 'article-item=test sports story'),
+    ).dblclick();
+
+    await expect(
+        page.locator(s('authoring', 'authoring-field=body_html')).getByRole('textbox'),
+    ).toHaveText('test sport story body');
+
+    await page.locator(s('Embed')).getByRole('button').click();
+
+    await page.getByPlaceholder('Enter URL or code to embed').type('https://google.com');
```

Comment:
How about using some fake url to avoid mentioning other companies/brands? Or let's at least adversite ourselves  `.type('https://superdesk.org')`